metron-deployment/roles/monit/tasks/scripts.yml

```diff
@@ -0,0 +1,53 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+---
+- name: Create monit home
+  file: path={{ monit_home }} state=directory mode=0755
+
+- name: Deploy parser topology start scripts
+  template:
+    src: scripts/start_topology.sh
+    dest: "{{ monit_home }}/start_{{ item }}_topology.sh"
+    mode: 0755
+  with_items:
+    - yaf
+    - snort
+    - bro
```

Comment:
I had to create custom start/stop/status scripts for the Storm topologies because Monit cannot handle passing an argument to a script.  Otherwise, I would have just used the ones we already have in `/usr/metron/[version]/bin`